jdaviz/configs/default/plugins/viewers.py

```diff
@@ -754,26 +756,41 @@ def set_plot_axes(self):
             u.bol, u.AB, u.ST
         ]
 
-        locally_defined_sb_units = [
-            unit / u.sr for unit in locally_defined_flux_units
-            ]
-
-        if any(y_unit.is_equivalent(unit) for unit in locally_defined_sb_units):
-            flux_unit_type = "Surface Brightness"
-        elif any(y_unit.is_equivalent(unit) for unit in locally_defined_flux_units):
-            flux_unit_type = 'Flux'
-        elif (
-                y_unit.is_equivalent(u.DN) or
-                y_unit.is_equivalent(u.electron / u.s) or
-                y_unit.physical_type == 'dimensionless'
-        ):
-            # electron / s or 'dimensionless_unscaled' should be labeled counts
-            flux_unit_type = "Counts"
-        elif y_unit.is_equivalent(u.W):
-            flux_unit_type = "Luminosity"
+        # get square angle from 'sb' display unit
+        sb_unit = self.jdaviz_app._get_display_unit(axis='sb')
+        if sb_unit is not None:
+            solid_angle_unit = check_if_unit_is_per_solid_angle(sb_unit, return_unit=True)
         else:
-            # default to Flux Density for flux density or uncaught types
-            flux_unit_type = "Flux density"
+            solid_angle_unit = None
```

Comment:
can we just access the angle display unit directly since unit conversion already contains this logic?

Reply:
I can't get angle directly from _get_display_unit, the only 'axis' that has angle information is surface brightness.

Reply:
discussed on slack, i will add an axis=angle to get_display_unit in a follow up PR